src/index.js

```diff
@@ -15,29 +14,67 @@ const choices = [
   "Create a LinkFree JSON file",
   "Update an existing JSON file",
   "Provide a testimonial to a LinkFree user",
-  "Add a given testimonial to your JSON file",
+  "Add an event",
 ];
 
 prompt([
   {
     type: "select",
     name: "selectedtask",
     choices: choices,
-    message: "Choose an icon (Press down arrow to see more options)",
+    message: "Choose one option (Press down arrow to traverse the list)",
   },
 ])
-  .then((answers) => {
+  .then(async (answers) => {
     const { selectedtask } = answers;
-    if (selectedtask === "Create a LinkFree JSON file") {
-      createJson();
-    } else if (selectedtask === "Update an existing JSON file") {
-      updateJson();
-    } else if (selectedtask === "Provide a testimonial to a LinkFree user") {
-      addTestimonial();
-    } else if (selectedtask === "Add a given testimonial to your JSON file") {
-      createTestimonial();
+    switch (selectedtask) {
+      case "Create a LinkFree JSON file": {
+        createJson();
+        break;
+      }
+      case "Update an existing JSON file": {
+        update();
+        break;
+      }
+      case "Provide a testimonial to a LinkFree user": {
+        giveTestimonial();
+        break;
+      }
+      default:
+        addEvent();
     }
   })
   .catch((error) => {
     console.error(error);
   });
+
+const update = async () => {
```

Comment:
Can we put both functions below in the `src/updatejson/updateJson.js`?
- githubUsername and getUsername

Reply:
> Can we put both functions below in the `src/updatejson/updateJson.js`? to make the index file clean
> - githubUsername and getUsername

I can do that

Reply:
Nice. we can also create a helper method inside the update folder